src/functions.php

```diff
@@ -2,6 +2,10 @@
 
 namespace React\Promise;
 
+/**
+ * @param mixed|null $promiseOrValue
+ * @return PromiseInterface
+ */
```

Comment:
I think it makes sense to include docblocks on our public API. However, if we do so, we should *carefully* follow what's documented in our README.